lib/MetaCPAN/Web/Controller/Sysadmin/Datacenters.pm

```diff
@@ -0,0 +1,29 @@
+package MetaCPAN::Web::Controller::Sysadmin::Datacenters;
+
+use Moose;
```

Comment:
MC::Moose sucks... you have to `use strict` and `use warnings` for perl critic tests